Resources/Shaders/InvertMaterial.gdshader

```diff
@@ -0,0 +1,14 @@
+shader_type spatial;
+render_mode unshaded;
+
+#include "res://Resources/ShaderIncs/ResoniteBlendMode.gdshaderinc"
+#include "res://Resources/ShaderIncs/ResoniteZTestMode.gdshaderinc"
+#include "res://Resources/ShaderIncs/ResoniteIncludes.gdshaderinc"
+
+uniform float Lerp : hint_range(0.0, 1.0) = 1.0;
+uniform sampler2D ScreenTexture : hint_screen_texture, filter_linear_mipmap;
```

Comment:
...unfortunately, Unity is complete trash at actually making it obvious what shader parameters are actually being used in the shader, so there's a few things you missed.
Firstly, even though it isn't implemented yet, you should go ahead and add ``_OffsetFactor`` and ``_OffsetUnits`` floats.
Secondly, this shader implements Rect clipping. I forgot to explain how to do it in my writing shaders document, but first, you need to add a vec4 uniform named ``_Rect``, next there's three macros you need to define in specific places, ``UI_Unlit`` has these in the right places.